src/parser.rs

```diff
@@ -444,28 +444,24 @@ impl Parser {
         };
         let on_overflow = if self.parse_keywords(vec!["ON", "OVERFLOW"]) {
             let error = self.parse_keyword("ERROR");
```

Comment:
You don't need the variable anymore. Simply: `if self.parse_keyword("ERROR") {`.